src/util/dates.js

```diff
@@ -140,3 +140,11 @@ export const formatDate = (intl, todayString, d) => {
 
   return `${formattedDate}, ${formattedTime}`;
 };
+
```

Comment:
A comment for these ISO8601 functions would be nice. Like the ones on other helper functions in this file.